File: src/main/java/net/dontdrinkandroot/gitki/wicket/headeritem/HighlightInitHeaderItem.java
package net.dontdrinkandroot.gitki.wicket.headeritem;

import org.apache.wicket.markup.head.HeaderItem;
import org.apache.wicket.markup.head.JavaScriptContentHeaderItem;

import java.util.Arrays;
import java.util.List;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class HighlightInitHeaderItem extends JavaScriptContentHeaderItem
{
    public HighlightInitHeaderItem()
    {
        super("hljs.initHighlightingOnLoad();", "highlight-init");
    }

    @Override
    public List<HeaderItem> getDependencies()
    {
        return Arrays.asList(
                new HightlightCssHeaderItem(),
                new HighlightJsHeaderItem()
        );
    }
}

File: src/main/kotlin/net/dontdrinkandroot/gitki/wicket/headeritem/HighlightInitHeaderItem.kt
package net.dontdrinkandroot.gitki.wicket.headeritem

import org.apache.wicket.markup.head.HeaderItem
import org.apache.wicket.markup.head.JavaScriptContentHeaderItem

class HighlightInitHeaderItem : JavaScriptContentHeaderItem("hljs.initHighlightingOnLoad();", "highlight-init") {

    override fun getDependencies(): List<HeaderItem> {
        return listOf(
            HightlightCssHeaderItem(),
            HighlightJsHeaderItem()
        )
    }
}